cb-talent-development/employer-style-base
# CE-496: 2017 CB Brand Gradient

**Purpose**
> As a user, I should see the same gradient used within products as I see on the website, so I feel connected all the way through my history with CareerBuilder - from buying to using.

This PR introduces a new gradient, `cbGradient2017` and replaces a few solid instances of it in ESB.

**JIRA**
https://careerbuilder.atlassian.net/browse/CE-496

**Changes**
* Improvements and fixes
  * New brand gradient introduced for the site

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * There will be unknowns. Some pages may display this new gradient and have bad copy. If you see this, please let me know in this PR.

**Screenshots**
* Before
<img width="1576" alt="screen shot 2017-11-10 at 3 27 23 pm" src="https://user-images.githubusercontent.com/5348098/32679339-bdea8b28-c62b-11e7-88ca-14ddc959f7a5.png">

* After
<img width="1626" alt="screen shot 2017-11-10 at 3 27 45 pm" src="https://user-images.githubusercontent.com/5348098/32679347-c43eb198-c62b-11e7-8e7f-e38175e3627d.png">

**Feature Server**
http://web.employer-6.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-6.development.c66.me/

* Steps to take 
For the following pages, verify that the indicated sections look alright. Verify that the gradient has the correct colors and is in the correct direction
  * Global - email subscribe footer
  * Homepage - verify the header still looks ok
  * /recruiting-solutions/small-business-subscription-plans-and-pricing
     * CTA section
     * Mobile pricing plan table headers
  * /recruiting-solutions/talent-discovery
     * Best Fit For section
  * /recruiting-solutions/talent-network
     * Best Fit For section
     * Robust Analytics - Product Feature Modal
  * /recruiting-solutions/applicant-tracking
     * Best Fit For section
  * /employment-screening
     * “Topped ranked in HRO Today’s…” section
  * /promotions
     * Hero and product table (if shown)
  * 404 Page
  * DY banners (should be overridden by another gradient iirc)
  * Go to the sitemap and look at some of the pages there as some use our generic deluxe template and had their headers updated (like /human-capital-management)

* Responsive considerations
  * There were no changes done to the affect sections beyond their gradients being switched out so you should not notice anything different on mobile

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/1041

**Additional Information**
N/A
